pkg/csi/service/wcpguest/controller_test.go

```diff
@@ -110,80 +116,85 @@ func createVolume(ctx context.Context, ct *controllerTest, reqCreate *csi.Create
  */
 func TestGuestClusterControllerFlow(t *testing.T) {
 	ct := getControllerTest(t)
+	modes := []csi.VolumeCapability_AccessMode_Mode{
+		csi.VolumeCapability_AccessMode_SINGLE_NODE_WRITER,
+		csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER,
+	}
+	for _, mode := range modes {
```

Comment:
Only changes from here onwards is to set `capabilities. AccessMode.Mode` to `mode`. Everything else has been indented since it was added to this for loop